src/workerd/api/worker-rpc.c++

```diff
@@ -153,80 +208,151 @@ kj::Maybe<JsRpcCapability::RpcFunction> JsRpcCapability::getRpcMethod(
   }));
 }
 
-// The capability that lets us call remote methods over RPC.
-// The client capability is dropped after each callRequest().
-class JsRpcTargetImpl final : public rpc::JsRpcTarget::Server {
+void JsRpcStub::serialize(jsg::Lock& js, jsg::Serializer& serializer) {
+  auto& handler = JSG_REQUIRE_NONNULL(serializer.getExternalHandler(), DOMDataCloneError,
```

Comment:
side note: In general, I think `DOMDataCloneError` is the only error type we should be throwing from these. I wonder if there's a reasonable way of enforcing that..

Reply:
Hmm, I'm not sure I could say confidently that all serialization errors are DataCloneError. Probably not worth trying to enforce.